test/unit/media.html5.js

```diff
@@ -71,6 +71,19 @@ test('test playbackRate', function() {
   strictEqual(tech.playbackRate(), 0.75);
 });
 
+test('should remove the controls attribute when recreating the element', function() {
```

Comment:
Yeah, I believe we only tested on 8.x. I don't know if we have a 7.x device test, but either way, it's how it works now. I did end up adding and IS_IPHONE check, but that's just a bandaid and I don't know how that really applies to what we're trying to test there. If anything else relies on that to be false it could be an issue.